Runtime 6.2.0 and 5.2.0 release notes (#1805)
astronomer/docs: astro/runtime-release-notes.md

### Additional improvements

> - Upgraded `astronomer-providers` to 1.14.0, which includes support for using a role ARN with `AwsBaseHookAsync`. See the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.13.0/CHANGELOG.rst) for a complete list of changes.

Reviewer: ```suggestion
- Upgraded `astronomer-providers` to 1.14.0, which includes support for using a role Amazon Resource Name (ARN) with `AwsBaseHookAsync`. See the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.13.0/CHANGELOG.rst) for a complete list of changes.
```

- Upgraded `astronomer-providers` to 1.11.2, which includes a collection of bug fixes. See the [`astronomer-providers` changelog](https://github.com/astronomer/astronomer-providers/blob/main/CHANGELOG.rst#1112-2022-11-19). 
- Upgraded `openlineage-airflow` to 0.17.0, which includes improvements to the OpenLineage spark integration and additional facets for the OpenLineage Python client. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/releases/tag/0.17.0) for more information.  

## Astro Runtime 6.2.0

- Release date: January 26, 2023

> - Airflow version: 2.4.3

Reviewer: ```suggestion
- Apache Airflow version: 2.4.3
```

> ### Early access Airflow bug fixes

Reviewer: ```suggestion
### Early access Apache Airflow bug fixes
```

> In anticipation of future support for the Kubernetes executor on Astro, Astro Runtime includes the following bug fixes from Airflow 2.5.2:

Reviewer: ```suggestion
In anticipation of future support for the Kubernetes executor on Astro, Astro Runtime includes the following bug fixes from Apache Airflow 2.5.2:
```

- Fix bad pods pickled in executor_config ([28454](https://github.com/apache/airflow/pull/28454))
- Be more selective when adopting pods with KubernetesExecutor ([28899](https://github.com/apache/airflow/pull/28899))

> - Only patch single label when adopting pod ([28776](https://github.com/apache/airflow/pull/28776))

Reviewer: ```suggestion
- Only patch single label when adopting pod ([28776](https://github.com/apache/airflow/pull/28776))
- Don’t re-patch pods that are already controlled by current worker ([26778](https://github.com/apache/airflow/pull/26778))
- Fix backfill queued task getting reset to scheduled state ([23720](https://github.com/apache/airflow/pull/23720))
```

### Additional improvements 

- Upgraded `astronomer-providers` to 1.14.0, which includes support for using a role ARN with `AwsBaseHookAsync`. See the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.13.0/CHANGELOG.rst) for a complete list of changes.

> - Upgraded `openlineage-airflow` to 0.19.2, which includes new support for Airflow operators like the `S3FileTransformOperator` and additional facets for task runs. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/releases/tag/0.19.2) for a complete list of changes.

Reviewer: ```suggestion
- Upgraded `openlineage-airflow` to 0.19.2, which includes new support for Airflow operators such as the `S3FileTransformOperator` and additional facets for task runs. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/releases/tag/0.19.2) for a complete list of changes.
```

- Upgraded `astronomer-providers` to 1.9.0, which includes two new deferrable versions of the operators from the dbt provider package. See the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.9.0/CHANGELOG.rst).
- Upgraded `openlineage-airflow` to version `0.14.1`. See the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).

## Astro Runtime 5.2.0

- Release date: January 26, 2023

> - Airflow version: 2.3.4

Reviewer: ```suggestion
- Apache Airflow version: 2.3.4
```

Reviewer: Is 2.3.4 correct?

> ### Early access Airflow bug fixes

Reviewer: ```suggestion
### Early access Apache Airflow bug fixes
```

> In anticipation of future support for the Kubernetes executor on Astro, Astro Runtime includes the following bug fixes from Airflow 2.5.2:

Reviewer: ```suggestion
In anticipation of future support for the Kubernetes executor on Astro, Astro Runtime includes the following bug fixes from Apache Airflow 2.5.2:
```
